gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/util/preflight/Preflight.kt

```diff
@@ -4,15 +4,15 @@ import com.emergetools.android.gradle.util.TreePrinter
 import org.gradle.api.GradleException
 import org.gradle.api.logging.Logger
 
-sealed class Result<T> {
-  class Success<T>(value: T) : Result<T>()
+sealed class Result {
+  object Success : Result()
```

Comment:
value was always of type `Unit` and was unused so I refactored this code to get rid of the unused warning.